step-templates/sql-deploy-dacpac-package-variable.json

```diff
@@ -0,0 +1,198 @@
\")\"\n        $filePath = $filePath[0]\n    }\n    elseif(@($filePath).Length -lt 1 -or $null -eq $filePath){\n        Throw \"Could not find $FileName.\"\n    }\n\n    return $filePath\n}\n\n# Define PowerShell Modules path\n$LocalModules = (New-Item \"$PSScriptRoot\\Modules\" -ItemType Directory -Force).FullName\n$env:PSModulePath = \"$LocalModules;$env:PSModulePath\"\n\n# Get the supplied parameters\n$PublishProfile = $OctopusParameters[\"DACPACPublishProfile\"]\n$Report = Format-OctopusArgument -Value $OctopusParameters[\"Report\"]\n$Script = Format-OctopusArgument -Value $OctopusParameters[\"Script\"]\n$Deploy = Format-OctopusArgument -Value $OctopusParameters[\"Deploy\"]\n$ExtractTargetDatabaseDacpac = Format-OctopusArgument -Value $OctopusParameters[\"ExtractTargetDatabaseDacPac\"]\n$TargetServer = $OctopusParameters[\"TargetServer\"]\n$TargetDatabase = $OctopusParameters[\"TargetDatabase\"]\n$TargetDatabaseVersion = $OctopusParameters[\"TargetDatabaseVersion\"]\n$UseIntegratedSecurity = Format-OctopusArgument -Value $OctopusParameters[\"UseIntegratedSecurity\"]\n$Username = $OctopusParameters[\"SQLUsername\"]\n$Password = $OctopusParameters[\"SQLPassword\"]\n$EnableMultiSubnetFailover = Format-OctopusArgument -Value $OctopusParameters[\"EnableMultiSubnetFailover\"]\n$AdditionalDeploymentContributors = Format-OctopusArgument -Value $OctopusParameters[\"AdditionalContributors\"]\n$AdditionalDeploymentContributorArguments = Format-OctopusArgument -Value $OctopusParameters[\"AdditionalContributorArguments\"]\n$PackageReferenceName = \"DACPACPackage\"\n\n$ExtractPathKey = (\"Octopus.Action.Package[{0}].ExtractedPath\" -f $PackageReferenceName)\n$ExtractPath = $OctopusParameters[$ExtractPathKey]\n\n\nif(!(Test-Path $ExtractPath)) {\n    Throw (\"The package extraction folder '{0}' does not exist or the Octopus Tentacle does not have permission to access it.\" -f $ExtractPath)\n}\n\n\n#$DACPACPackagePath = Get-DacFilePath -BasePath $ExtractPath -FileName $DACPACPackageName -FileType \"DacPac\"\n$DACPACPackagePath = Get-DacFilePath -BasePath $ExtractPath -FileName $DACPACPackageName -FileType \"DacPac\"\n\nIf ($PublishProfile){\n    $PublishProfilePath = Get-DacFilePath -BasePath $ExtractPath -FileName $PublishProfile -FileType \"PublishProfile\"\n}\n\n# Invoke the DacPac utility\ntry\n{\n    Invoke-DacPacUtility -Report $Report -Script $Script -Deploy $Deploy -ExtractTargetDatabaseDacpac $ExtractTargetDatabaseDacpac -DacPacFilename $DACPACPackagePath -TargetServer $TargetServer -TargetDatabase $TargetDatabase -UseIntegratedSecurity $UseIntegratedSecurity -Username $Username -Password $Password -EnableMultiSubnetFailover $EnableMultiSubnetFailover -PublishProfile $PublishProfilePath -AdditionalDeploymentContributors $AdditionalDeploymentContributors -AdditionalDeploymentContributorArguments $AdditionalDeploymentContributorArguments -DataToolsExtractPath $DataToolsExtractPath\n}\ncatch\n{\n    Write-Host $_.Exception.ToString()\n    throw;\n}\n",
```

Comment:
As per https://github.com/OctopusDeploy/Library/pull/886#issuecomment-600320909 can you remove support for TLS 1.1?